bin/truffleruby

```diff
@@ -120,7 +138,28 @@ if [ -n "$CP" ]; then
 fi
 
 declare -a full_command
-full_command=("$JAVACMD" $JAVA_OPTS "-Xbootclasspath/a:$boot_cp_jar" "${java_args[@]}" org.truffleruby.Main "-Xhome=$root" "-Xlauncher=$root/bin/truffleruby" "${ruby_args[@]}")
+if [ $on_graalvm = false ]; then
+    full_command=(
+        "$JAVACMD"
+        $JAVA_OPTS
+        "-Xbootclasspath/a:$boot_cp_jar"
+        "${java_args[@]}"
+        org.truffleruby.Main
+        "-Xhome=$root"
+        "-Xlauncher=$root/bin/truffleruby"
+        "${ruby_args[@]}"
```

Comment:
And the whole purpose of this is to make language/ruby look like a normal Ruby home, so it only makes sense.